cmd/comparator-rest/docs/openapi.yaml

```diff
@@ -69,43 +69,20 @@ paths:
     post:
       tags:
         - required
+      description: Evaluates an operator with its inputs and returns the result.
       requestBody:
         required: true
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/Comparison"
-            example: {
-              "op": {
-                "type": "eq",
-                "args": [
-                  {
-                    "type": "doc",
-                    "docID": "did:example:123",
-                    "authTokens": {
-                      "edv": "21tDAKCERh95uGgKbJNHYp",
-                      "kms": "bcehfew7h32f32h7af3"
-                    }
-                  },
-                  {
-                    "type": "auth",
-                    "authToken": "123456789abcdefghi"
-                  }
-                ]
-              }
-            }
+              $ref: "#/components/schemas/ComparisonRequest"
```

Comment:
There doesn't seem to be any practical need for this refactor.

Reply:
done